armi/reactor/blocks.py

```diff
@@ -176,42 +177,6 @@ def core(self):
         c = self.getAncestor(lambda c: isinstance(c, Core))
         return c
```

Comment:
It's still weird that we have `Block.core`, but fine. That's not for this PR.

Also... there is no error handling on `Block.core`? Mark that as sus.

Reply:
What sort of error handling do you have in mind?

Reply:
What if the `Block` is not part of a `Core`?

You know, when you're building a `Reactor` from `Blueprints`, you have to create a `Block` and add it to the `Core`.  Or you might be doing some fuel shuffling, and not all `Assemblies`/`Blocks` are in the `Core` at the same time.  Or your `Block` might be in the `SpentFuelPool`.  Lots of situations.

But the question is... should that return `None`, or `raise Error()` or what?  Hmm... What works, but isn't heavy handed?  hmmm

Reply:
Hmmmm. Well `getAncestor` currently returns `None`. I suppose for the case where it's a SFP or something else, I could see a runLog or something being useful. 

Though resolving this feels like worthy of a separate PR. What do you think?